internal/driver/nodeserver.go

```diff
@@ -140,24 +140,25 @@ func (ns *NodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 func (ns *NodeServer) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpublishVolumeRequest) (*csi.NodeUnpublishVolumeResponse, error) {
 	log, ctx, done := logger.GetLogger(ctx).WithMethod("NodeUnpublishVolume")
 	defer done()
-	
+
 	volumeID := req.GetVolumeId()
-	targetPath := req.GetTargetPath()
-	log.V(2).Info("Processing request", "volumeID", volumeID, "targetPath", targetPath)
-	
+	log.V(2).Info("Processing request", "volumeID", volumeID)
+
 	ns.mux.Lock()
 	defer ns.mux.Unlock()
 
 	// Validate request object
 	log.V(4).Info("Validating request", "volumeID", volumeID)
-	if err := validateNodeUnpublishVolumeRequest(ctx, req); err != nil {
+	err := validateNodeUnpublishVolumeRequest(ctx, req)
+	if err != nil {
 		return nil, err
 	}
```

Comment:
nit: Can we use this pattern if a func is returning back just an err?

`if err := validateNodeUnpublishVolumeRequest(ctx, req); err != nil {
      return nil, err
}`

Reply:
agree. done